coral-trino/src/test/java/com/linkedin/coral/trino/rel2trino/HiveToTrinoConverterTest.java

```diff
@@ -754,9 +780,20 @@ public void testRlikeTransformation() {
   public void testRegexpTransformation() {
     RelToTrinoConverter relToTrinoConverter = new RelToTrinoConverter();
 
-    RelNode relNode = hiveToRelConverter.convertSql("SELECT '1' NOT REGEXP '^1$'");
+    RelNode relNode = TestUtils.getHiveToRelConverter().convertSql("SELECT '1' NOT REGEXP '^1$'");
     String targetSql = "SELECT NOT \"REGEXP_LIKE\"('1', '^1$')\n" + "FROM (VALUES  (0)) AS \"t\" (\"ZERO\")";
     String expandedSql = relToTrinoConverter.convert(relNode);
     assertEquals(expandedSql, targetSql);
   }
+
+  public void testSqlSelectAliasAppenderTransformer() {
+    //test.tableA(a int, b struct<b1:string>
```

Comment:
nit: a space after `//`, or we can remove this comment since there is no similar comment for other UTs.

Reply:
I can fix it in the next PR if its not a blocking concern. Thanks